OsmAnd/src/net/osmand/plus/views/MapControlsLayer.java

```diff
@@ -125,6 +125,13 @@ public void initLayer(final OsmandMapTileView view) {
 		initTransparencyBar(view, parent);
 	}
 
+	public void startNavigation() {
```

Comment:
doesn't look like a good solution... though it does exactly as we press on the button, that doesn't the code should be in the button. Better place to extract from the button and put it somewhere MapActivityActions. And we don't go from MapActivity to a specific layer